src/Components/Categories/CategoriesForm.js

```diff
@@ -1,32 +1,213 @@
-import React, { useState } from 'react';
```

Comment:
this changes are not scope of this user story, please delete this file from this PR

Reply:
Done!